grammar punctuation mostly

annekz opened the pull request:
I made a bunch of minor edits to the text

annekz committed:
grammar punctuation mostly

DiegoPino commented on docs/giveortake.md:
```diff
@@ -10,19 +10,19 @@ We will be helpful, kind, and open. We encourage discussions and always respect
+  What? Code: Deals with exposing and transforming the JSON when navigating the site. What is displayed, how it is displayed. Provides templating, metadata display entities via Twig, and direct file downloads. In its Drupal form it provides many field formatters for Strawberryfield` and a content entity for the Twig templates.
 - [Archipelago Deployment](https://github.com/esmero/archipelago-deployment/tree/8.x-1.0-beta1)
-  What? DevOps: Docker-compose deployment strategy, including a full skeleton projects with persistance folders for Min.io, DB, Solr, Cantaloupe and Drupal 8. Includes initial deployment configurations, what modules are enabled, how things look in Drupal 8 and some scripts plus the deployment documentation for both OSX and Linux.
+  What? DevOps: Docker-compose deployment strategy, including a full skeleton of projects with persistence folders for Min.io, DB, Solr, Cantaloupe and Drupal 8. Includes initial deployment configurations, which modules are enabled, how things look in Drupal 8 and some scripts plus the deployment documentation for both OSX and Linux.
```
This edit needs a change ,but i understand the why of your suggestion. The tech term is `a skeleton project` and that needs to stay (means a project with just the main structure and not all the packages), so even when 
> including a full skeleton of projects with

is correct english, its not what we deliver.

DiegoPino commented on docs/giveortake.md:
```diff
@@ -10,19 +10,19 @@ We will be helpful, kind, and open. We encourage discussions and always respect
 ## Questions, Answers and In Transit Between Both Ends
 
-We host a community interaction channel, our [google group](https://groups.google.com/forum/#!forum/archipelago-commons). This is the best place to ask questions and make suggestions that are not specific to a single module and/or you think can contribute to a larger discussion within our community. Discussions work best in this forum (not excluding GitHub of course), and our official announcemen ...
+We host a community interaction channel, our [google group](https://groups.google.com/forum/#!forum/archipelago-commons). This is the best place to ask questions and make suggestions that are not specific to a single module, and/or if you would like to contribute to a larger conversation within our community. Discussions work best in this forum (not excluding GitHub of course), and our official a ...
```
I agree, this reads better.

DiegoPino reviewed (changes requested):
There is a little change request here, all the rest looks awesome. Please let me know if you can address that comment to approve and merge. Thanks for your contribution!

annekz committed:
grammar etc correction - skeleton project

I changed it to "a full skeleton project" - (instead of a full skeleton projects). Let me know if that's right. Also hope I'm doing this right without Sarah over my shoulder... :)

DiegoPino reviewed (approved):
Great! Approving
